tests/unit_tests/test_wave_06.py

```diff
@@ -76,16 +76,19 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
     # Assertions should check:
-    # - That the results is truthy
-    # - That tai and jesse's inventories are the correct length
-    # - That all the correct items are in tai and jesse's inventories, including the items which were swapped from one vendor to the other
+    assert result == True
+    
+    assert len(tai.inventory)==3
+    assert len(jesse.inventory)==3
+    assert item_f in tai.inventory
+    assert item_c in jesse.inventory
```

Comment:
👍

Reply:
Make sure you add spaces between names, operators, and values as according to pep8 guidelines:

```python
    assert result == True
    assert len(tai.inventory) == 3
    assert len(jesse.inventory) == 3
    assert item_f in tai.inventory
    assert item_c in jesse.inventory
```